docs/api-reference/webgl/framebuffer.md

```diff
@@ -68,7 +68,6 @@ model.draw({
 
 Binding a framebuffer for multiple render calls
 ```js
-import {withState} from 'luma.gl';
```

Comment:
Change to `import {withParameters}`?

Reply:
actually `withState` was not used in this example.